web/web/api/v1/__init__.py

```diff
@@ -0,0 +1,3 @@
+from web import app
+#enables trailing and non-trailing slash routes
+app.url_map.strict_slashes = False
```

Comment:
let's put this on the `web/web/__init__.py` instead. That way if/when we create a public api or an api v2 we won't have to worry about remembering to set this again.

Reply:
Ok will do